packages/squiggle-lang/src/rescript/Reducer/Reducer_Dispatch/Reducer_Dispatch_BuiltIn.res

```diff
@@ -129,6 +129,23 @@ let callInternal = (call: functionCall, environment, reducer: ExpressionT.reduce
     )
   }
 
+  let doValueOfUnit = (aUnit: string) =>
+    switch aUnit {
+    | "nm" => 0.000000001->EvNumber->Ok
+    | "um" => 0.000001->EvNumber->Ok
+    | "mm" => 0.001->EvNumber->Ok
+    | "cm" => 0.01->EvNumber->Ok
+    | "dm" => 0.1->EvNumber->Ok
+    | "m" => 1.->EvNumber->Ok
+    | "dam" => 10.->EvNumber->Ok
+    | "hm" => 100.->EvNumber->Ok
+    | "km" => 1000.->EvNumber->Ok
+    | "Mm" => 1000000.->EvNumber->Ok
+    | "Gm" => 1000000000.->EvNumber->Ok
+    | "Tm" => 1000000000000.->EvNumber->Ok
```

Comment:
This is a great usecase for scientific notation, so that people don't have to count the zeroes.